server/app.js

```diff
@@ -180,16 +178,16 @@ passport.use(
       passwordField: config.auth.passwordField,
       passReqToCallback: true,
     },
-    function (req, username, password, done) {
-      mongodb
-        .collection('users')
-        .findOne({ uid: username })
-        .then(function (err, user) {
-          if (!user) {
-            return done(null, false, req.body)
-          }
-          return done(null, true, null)
+    async function (req, username, password, done) {
+      try {
+        const user = await prisma.users.findFirst({
+          where: { uid: username },
         })
+        if (!user) return done(null, false, req.body)
+        else done(null, true, null)
```

Comment:
do we need to explicitly return here?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/237#:~:text=%E2%80%A6-,7930718,-Merge%20state